<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AlgGuiClass</class>
 <widget class="QDialog" name="AlgGuiClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1353</width>
    <height>616</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="maximumSize">
   <size>
    <width>1920</width>
    <height>1080</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>AlgGui</string>
  </property>
  <widget class="QGroupBox" name="groupBox_3">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>10</y>
     <width>961</width>
     <height>581</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>451</width>
      <height>281</height>
     </rect>
    </property>
    <property name="title">
     <string>Frame</string>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="frameLabel">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>10</y>
      <width>451</width>
      <height>281</height>
     </rect>
    </property>
    <property name="title">
     <string>Background</string>
    </property>
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="0" column="0">
      <widget class="QLabel" name="bgLabel">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QGroupBox" name="groupBox_5">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>290</y>
      <width>451</width>
      <height>281</height>
     </rect>
    </property>
    <property name="title">
     <string>FGMask</string>
    </property>
    <layout class="QGridLayout" name="gridLayout_3">
     <item row="0" column="0">
      <widget class="QLabel" name="fgLabel">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QGroupBox" name="groupBox_6">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>290</y>
      <width>451</width>
      <height>281</height>
     </rect>
    </property>
    <property name="title">
     <string>TSOEvidence</string>
    </property>
    <layout class="QGridLayout" name="gridLayout_4">
     <item row="0" column="0">
      <widget class="QLabel" name="tsoLabel">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>30</y>
     <width>301</width>
     <height>61</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_4">
    <item>
     <widget class="QPushButton" name="BtnFilePath">
      <property name="text">
       <string>Open File..</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="BtnCloseVideo">
      <property name="text">
       <string>Close</string>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="horizontalSpacer_2">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QPushButton" name="BtnExit">
      <property name="text">
       <string>Exit</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QGroupBox" name="paramGroupBox">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>100</y>
     <width>301</width>
     <height>141</height>
    </rect>
   </property>
   <property name="title">
    <string>Param</string>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_6">
      <item>
       <widget class="QLabel" name="label">
        <property name="font">
         <font>
          <family>宋体</family>
          <pointsize>11</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Resize方式:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="BoxOfResizeWay">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="currentText">
         <string>INTER_AREA</string>
        </property>
        <property name="currentIndex">
         <number>0</number>
        </property>
        <item>
         <property name="text">
          <string>INTER_AREA</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string notr="true">INTER_NEAREST</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label_2">
        <property name="font">
         <font>
          <pointsize>11</pointsize>
         </font>
        </property>
        <property name="text">
         <string>启动YOLO:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="BoxOfIsYolo">
        <item>
         <property name="text">
          <string>否</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>是</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="label_3">
        <property name="font">
         <font>
          <family>宋体</family>
          <pointsize>11</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Alarm时间：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="BoxOfAlarmTime">
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>999</number>
        </property>
        <property name="value">
         <number>120</number>
        </property>
        <property name="displayIntegerBase">
         <number>10</number>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="AlgGui.qrc"/>
 </resources>
 <connections/>
 <slots>
  <slot>on_chkBoxUnder_clicked(bool)</slot>
  <slot>on_chkBoxItalic_clicked(bool)</slot>
  <slot>on_chkBoxBold_clicked(bool)</slot>
  <slot>on_rBtnCam_clicked(bool)</slot>
  <slot>on_rBtnVidFile_clicked(bool)</slot>
 </slots>
</ui>
